app/helpers/course/discussion/topics_helper.rb

```diff
@@ -15,7 +15,8 @@ def topics_title
   # @param [Integer] line_end The one based end line line number.
   # @return [String] A HTML fragment containing the code lines.
   def display_code_lines(file, line_start, line_end)
-    code = file.lines((line_start - 1)..(line_end - 1)).join("\n")
+    code = file.add_line.lines((line_start - 1)..(line_end - 1)).join("\n")
```

Comment:
Could you add a comment above this line on why a line is added?

Reply:
Ok done!